index.html

```diff
@@ -48,32 +48,71 @@ <h2>Advancing the future of the internet</h2>
         <!-- Benefits section -->
         <section class="mzp-l-content benefits-section">
             <div class="benefits-text-wrapper">
-                <h2><span class="dark-highlight">Benefits & Perks</span></h2>
+                <h2><span class="dark-highlight">Why Join Us?</span></h2>
             </div>
-            <div class="mzp-t-picto-center">
+            <div class="why-join-container">
+                <h3 class="why-join-subheader">For Builders</h3>
                 <section class="mzp-c-picto">
                     <div class="mzp-c-picto-image">
-                        <img src="/img/handshake.svg" width="90" alt="">
+                        <img src="/img/handshake.svg" height="75" alt="">
                     </div>
                     <h3 class="mzp-c-picto-heading">No-strings attached grant – we don’t take equity!</h3>
                 </section>
-            </div>
-            <div class="mzp-t-picto-center">
                 <section class="mzp-c-picto">
                     <div class="mzp-c-picto-image">
-                        <img src="/img/lightbulb.svg" width="90" alt="">
+                        <img src="/img/lightbulb.svg" height="75" alt="">
                     </div>
                     <h3 class="mzp-c-picto-heading">Stipends for learning and networking (e.g. conferences)</h3>
                 </section>
-            </div>
-            <div class="mzp-t-picto-center">
                 <section class="mzp-c-picto">
                     <div class="mzp-c-picto-image">
-                        <img src="/img/world.svg" width="90" alt="">
+                        <img src="/img/world.svg" height="75" alt="">
                     </div>
                     <h3 class="mzp-c-picto-heading">Access to Mozilla’s ecosystem of people, partners, and organizations</h3>
                 </section>
-            </div>
+            </div> <!-- end of why us section -->
+            <div class="why-join-container">
+                <h3 class="why-join-subheader">For Collaborators</h3>
+                <section class="mzp-c-picto">
+                    <div class="mzp-c-picto-image">
+                        <img src="/img/grant.svg" height="75" alt="">
```

Comment:
**Suggestion:** So the page loaded with the images being big and overflowing out of their containers (even after hard-refreshing and re-running the server). I played around with the code for it to work, and when I undid my changes, the overflowing images didn't return. This tells me that there's some wonkiness happening here.

I suggest removing the `height` attribute off of these `img` tags within this section, and depending on the CSS file alone to size the image.